server/index.js

```diff
@@ -1,2 +1,52 @@
 const express = require("express");
+const axios = require("axios");
+
 const app = express();
+
+const TOKEN =
+    "13171~G4mR2rRs5D3lcWb83MIm0mxf2B0sy3NRQUi8wgUlnWKisJtGh7YrMGuR3w6uepHH";
```

Comment:
Don't expose your token. Use an environment variable or take in a token as a parameter